sdk/src/main/java/io/opentelemetry/sdk/metrics/aggregator/LongSumAggregator.java

```diff
@@ -16,14 +16,25 @@
 
 package io.opentelemetry.sdk.metrics.aggregator;
 
+import io.opentelemetry.sdk.metrics.data.MetricData.LongPoint;
+import io.opentelemetry.sdk.metrics.data.MetricData.Point;
+import java.util.Map;
 import java.util.concurrent.atomic.AtomicLong;
 
 public final class LongSumAggregator implements Aggregator {
+  private static final AggregatorFactory AGGREGATOR_FACTORY =
+      new AggregatorFactory() {
+        @Override
+        public Aggregator getAggregator() {
+          return new LongSumAggregator();
+        }
+      };
+
   // TODO: Change to use LongAdder when changed to java8.
```

Comment:
With #744 I guess this TODO can be removed

Reply:
LongAdder is from java8 not java7 (sad face)